torch/legacy/nn/Padding.py

```diff
@@ -22,6 +25,9 @@ def updateOutput(self, input):
         self.outputSize = torch.Size(outputSize)
         dim = self.dim
 
+        if self.nInputDim and input.dim() != self.nInputDim:
```

Comment:
Accessing `self.nInputDim` will raise an error (not return None like in Lua). You need to make sure it's set with `hasattr(self, 'nInputDim')`

Reply:
Ahh gotcha! Thanks @apaszke

Reply:
Just a quick check to make sure I'm not missing something.
I am setting the `nInputDim` variable in the constructor and it defaults to 0, hence in the forward pass it should evaluate to false. Is the current error a product of the testing framework, because I have never encountered such behavior  before.